README.md

```diff
@@ -1 +1,72 @@
-# Awesome-Books
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome-Books
+
+> week1 module 2 javascript project
```

Comment:
[OPTIONAL] To help others learn more about your project, it would be helpful to add a detailed description to your Readme.